README.md

```diff
@@ -5,7 +5,9 @@
   - [GitHub Docker Container Action Creation](#github-docker-container-action-creation)
   - [Invoking the Repository Workflow from another Repo](#invoking-the-repository-workflow-from-another-repo)
   - [Manually Invoking the Repository Workflow with GitHub App Authentication](#manually-invoking-the-repository-workflow-with-github-app-authentication)
-  - [Running the notebook manually](#running-the-notebook-manually)
+  - [Running the Notebook Manually](#running-the-notebook-manually)
+  - [Notebook Input Parameters](#notebook-input-parameters)
+  - [Installing maap-py](#installing-maap-py)
   - [References](#references)
 
 ## GitHub App Creation and Installation
```

Comment:
I'm a bit late to this party but I think this README is awesome but needs a bit more direction in the beginning - I think we have to use the github app so that other repositories (e.g. "fake-app" and "cmr") are authorized to trigger github worklows in this repository, but that's not stated here so it would be confusing to know what this Github App section is here for.

Reply:
I'll add something